be/src/olap/schema_change.cpp

```diff
@@ -1279,56 +1244,48 @@ OLAPStatus SchemaChangeHandler::_do_alter_tablet(
         for (int64_t transaction_id : transaction_ids) {
             if (!TxnManager::instance()->has_txn(
                 partition_id, transaction_id,
-                ref_tablet->tablet_id(), ref_tablet->schema_hash())) {
+                base_tablet->tablet_id(), base_tablet->schema_hash())) {
                 finished_transactions.push_back(transaction_id);
             }
         }
         for (int64_t transaction_id : finished_transactions) {
             transaction_ids.erase(transaction_id);
             VLOG(3) << "transaction finished when schema change is waiting. "
-                    << "tablet=" << ref_tablet->full_name()
+                    << "tablet=" << base_tablet->full_name()
                     << ", transaction_id=" << transaction_id
                     << ", transaction_size=" << transaction_ids.size();
         }
     }
 
     // 2. Get version_to_be_changed and store into tablet header
-    ref_tablet->obtain_push_lock();
-    ref_tablet->obtain_header_wrlock();
+    base_tablet->obtain_push_lock();
+    base_tablet->obtain_header_wrlock();
```

Comment:
this name should be rename to obtain_meta_wrlock?
header should not be used

Reply:
OK, this modification may refer to many codes, I will push it on a lonely commit.